bom/application/pom.xml

```
@@ -4738,6 +4739,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.apache.sshd</groupId>
+                <artifactId>sshd-common</artifactId>
+                <version>${sshd-common.version}</version>
+            </dependency>
```

Some Camel Quarkus extensions depend on both sshd-core and sshd-common. Hence it would be nice if Quarkus could manage both.

```suggestion
            <dependency>
                <groupId>org.apache.sshd</groupId>
                <artifactId>sshd-core</artifactId>
                <version>${sshd-common.version}</version>
            </dependency>
            <dependency>
                <groupId>org.apache.sshd</groupId>
                <artifactId>sshd-common</artifactId>
                <version>${sshd-common.version}</version>
            </dependency>
```